src/widgets/requirements/ui/requirements.test.tsx

```diff
@@ -1,8 +1,10 @@
 import { screen } from '@testing-library/react';
 import { beforeEach, describe, expect, it } from 'vitest';
-import { Requirements } from './ui/requirements';
+import { Requirements } from './requirements.tsx';
```

Comment:
```suggestion
import { Requirements } from './requirements';
```
Do we need to use extension .tsx here?

Reply:
Done